### DISCUSSION TITLE
🏳️‍🌈 👋 Hey to everyone 👋 🏳️‍🌈

### DISCUSSION THREAD

**Discussion by DevRehannaLP** [community/community#119688] [New to GitHub]
Hey everyone 😅 

I wish I had a story like so many I have read, to tell you what school I'm in and what skills or education I have.  But I've been dreaming of becoming a computer programmer since I was about 8.  I've done everything, almost in life so far, starting from working fast food at the age of 15 to becoming a magician, auto mechanic, military soldier & helicopter repairer, working in the medical field, becoming an indie pro wrestler, semi-truck driver, vocalist, and even lawn care maintenance.  It wasn't until my father came to me in my dream and stated how he loved hearing how I loved computers and how I wanted to be a programmer, but now he wonders why I am always in front of a computer nearly every day of my life this far and yet I haven't even followed my true dreams.  Oddly, that same day my mother, who's more ill than I care to admit, told me how proud I have made her, but she would want nothing more for me than to see me go after what I always talked about as far back as she could remember.

So, I started looking into it though starting anything at this point in my life would be most difficult.  Yet here I am.  Learning is one of the hardest things I've ever had to learn in my life... Code! 👨‍💻 

I can say though I've been doing this for a few weeks now, literally, all the training I have has been self-taught through studying what I can, and I have GitHub to thank a lot, for what I have learned and have been able to accomplish this far.  GitHub has been and is my educational system, support system, and mentor through this challenging part of my life.  I know I don't know enough of anything yet, but I do plan, God willing, to become an amazing coder, computer programmer, developer, DevOps, full-stack professional one day 😅 

P.S. I will take any advice, tips, guidance, anything that will help me advance along the way because um...I'm scared 😆 Most times I don't know what I'm doing lol

![Reginald Leon Portee](https://github.com/community/community/assets/164944951/1ca037c6-a05c-424f-a3bc-a916e851d0d9)

**Comment by ghostinhershell** [comment]
Welcome to GitHub and the GitHub Community @leonxpx 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!

**Comment by DevRehannaLP** [comment]
@ghostinhershell thank you for the warm welcome!  I have to admit in my short time coming into the world of GitHub, I've loved it, hated it, wanted to become more envolved with it and at the same time I've quit already 3 times 😆 but I keep coming back because as tasking as it may be I do love it 😏 .  I've gotten a little better with my profile butt still have a long way to go.  Thanks for your reply and welcome and I'm checking out the check in right now! 

Happy Coding!
